code/game/jobs/job/security.dm

```diff
@@ -1,5 +1,5 @@
 /datum/job/hos
-	title = "Head of Security"
+	title = "Head Security Chief"

```

Comment:
This is redundant. Chief and Head are functionally identical.
`Lead security leader`, essentially.